### PR TITLE
fix: 3D frame comparison

### PR DESCRIPTION
**Pull request by liuxi150**
## Summary by CodeRabbit

- **New Features**
  - Enhanced scene loading with additional parameters for better compatibility and dynamic sorting.
  - Introduced functionality for comparing and processing glTF frames in a web environment.

- **Improvements**
  - Improved animation and scene management with updated duration properties and enhanced camera settings.
  - Refined material properties for better handling of render types and face rendering.
  - Updated `TestPlayer` class with new initialization and resource management options.

- **Bug Fixes**
  - Fixed various issues in handling texture sources, animations, and skybox parameters to improve rendering accuracy.
  
- **Dependencies**
  - Updated `@vvfx/resource-detection` to version `0.6.0-alpha.9`.

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.0'
+lockfileVersion: '9.0'
```
Comment (yiiqii) [inline pnpm-lock.yaml:1]: lockfileVersion 就不要动了，看看是不是 pnpm 版本导致的

File: web-packages/test/case/3d/src/gltf/new-loader.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import { loadGLTFSceneECS } from '../../../../../../plugin-packages/model/src/gltf/loader-ecs-ext';
```
Comment (yiiqii) [inline web-packages/test/case/3d/src/gltf/new-loader.ts:1]: `import { loadGLTFSceneECS } from '@galacean/effects-plugin-model'`

**Review by yiiqii — CHANGES_REQUESTED** [review]

File: pnpm-lock.yaml (hunk 1 above)
Comment (liuxi150) [reply pnpm-lock.yaml]: 因为这次合并后要安装新的spec，我通过pnpm i的方式安装就修改了lock文件，我现在回退了到6.0版本。
我本地的pnpm是9.1.2

File: web-packages/test/case/3d/src/gltf/new-loader.ts (hunk 2 above)
Comment (liuxi150) [reply web-packages/test/case/3d/src/gltf/new-loader.ts]: loadGLTFSceneECS 这个函数没有导出来，因为3d插件不能直接以来resource-detection，所以我先迁移相关代码解决了。

File: pnpm-lock.yaml (hunk 1 above)
Comment (yiiqii) [reply pnpm-lock.yaml]: 不是手动改 pnpm-lock 文件。。我操作一下提交吧

**Review by yiiqii — APPROVED** [review]

**Pull request merged**